variants/rak4631_epaper/platformio.ini

```diff
@@ -20,3 +20,32 @@ lib_deps =
 debug_tool = jlink
 ; If not set we will default to uploading over serial (first it forces bootloader entry by talking 1200bps to cdcacm)
 ;upload_protocol = jlink
+
+[env:rak4631_eink_dynamic]
+extends = nrf52840_base
+board = wiscore_rak4631
+build_flags = ${nrf52840_base.build_flags} -Ivariants/rak4631_epaper -D RAK_4631_D
```

Comment:
If i recall correctly, the RAK_4631 macro is used in a few places in the firmware. Maybe define the RAK_4631_D macro in addition to RAK_4631

Reply:
yes, for a dynamicEink variant